### ISSUE TITLE
Plugin does not resolve font assets using resolve aliases

### ISSUE THREAD

**Issue by ElijahSink** [laynezh/vite-plugin-lib-assets#25]
I'm using version `0.5.4-beta.0`.

I have an SCSS file:
```scss
@font-face {
  font-family: 'VectoraLTStd';
  src: url('@Assets/fonts/interface/VectoraLTStd-Roman.woff2') format('woff2'), url('@Assets/fonts/interface/VectoraLTStd-Roman.woff') format('woff');
  font-weight: normal;
  font-style: normal;
}

@font-face {
  font-family: 'VectoraLTStd';
  src: url('@Assets/fonts/interface/VectoraLTStd-Bold.woff2') format('woff2'), url('@Assets/fonts/interface/VectoraLTStd-Bold.woff') format('woff');
  font-weight: bold;
  font-style: normal;
}
```

That's making use of a resolve alias as defined in my `vite.config.ts`:
```ts
...
resolve: {
      alias: [
        ...
        { find: "@Assets", replacement: fileURLToPath(new URL("./src/assets", import.meta.url)) },
...
```
This alias works fine in builds without the plugin, and the fonts are inlined in the output CSS as base64.

However, with the plugin, I get the following errors:
```
vite v4.4.9 building for production...
transforming (123) src/components/templateSelection/CategorySelectionButtons.vue
[vite-plugin-lib-assets]: file not found path-redacted/src/scss/@Assets/fonts/interface/VectoraLTStd-Roman.woff2
[vite-plugin-lib-assets]: file not found path-redacted/src/scss/@Assets/fonts/interface/VectoraLTStd-Roman.woff
[vite-plugin-lib-assets]: file not found path-redacted/src/scss/@Assets/fonts/interface/VectoraLTStd-Bold.woff2
[vite-plugin-lib-assets]: file not found path-redacted/src/scss/@Assets/fonts/interface/VectoraLTStd-Bold.woff
✓ 149 modules transformed.
✓ built in 3.71s
[commonjs--resolver] Could not set source for asset "assets/VectoraLTStd-Roman.[hash:8].woff2", asset source needs to be a string, Uint8Array or Buffer.
error during build:
RollupError: Could not set source for asset "assets/VectoraLTStd-Roman.[hash:8].woff2", asset source needs to be a string, Uint8Array or Buffer.
    at error (file:///path-redacted/node_modules/rollup/dist/es/shared/node-entry.js:2245:30)
    at getValidSource (file:///path-redactednode_modules/rollup/dist/es/shared/node-entry.js:24880:16)
    at FileEmitter.emitAsset (file:///path-redacted/node_modules/rollup/dist/es/shared/node-entry.js:25040:15)
    at FileEmitter.emitFile (file:///path-redacted/node_modules/rollup/dist/es/shared/node-entry.js:24920:25)
    at emitFile (file:///path-redacted/node_modules/@laynezh/vite-plugin-lib-assets/dist/index.js:65:13)
    at file:///path-redacted/node_modules/@laynezh/vite-plugin-lib-assets/dist/index.js:170:27
    at Array.forEach (<anonymous>)
    at Object.resolveId (file:///path-redacted/node_modules/@laynezh/vite-plugin-lib-assets/dist/index.js:169:21)
    at async PluginDriver.hookFirstAndGetPlugin (file:///path-redacted/node_modules/rollup/dist/es/shared/node-entry.js:25319:28)
    at async resolveId (file:///path-redacted/node_modules/rollup/dist/es/shared/node-entry.js:23993:26)
```

**Comment by ElijahSink** [comment]
The same resolve alias works fine for `.png` assets processed inside of `.vue` or `.ts` files--they are processed by the plugin correctly.

**Comment by coder-layne** [comment]
Thank you once again for your feedback. I'll check this problem later.

**Comment by coder-layne** [comment]
Sorry for the delay on response, I've just released v0.5.4-beta.1, which now supports aliases in style files.
